src/components/Header/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import bookOpenIcon from './BookOpen.svg';
+import bookmarkIcon from './BOOKMARK_SIMPLE.svg';
+
+class Header extends Component {
+  render() {
+    return (
+      <header className="wrapper pt-5 pb-5 flex justify-between">
+        <div className="flex">
+          <img className="pr-3" src={bookOpenIcon} alt="book icon" />
+          <h3>Open Library</h3>
```

Comment:
It should be `<h1>` I think. Probably it is the main title of the page. If not, as it is a logo, it just can be a `<span>`

Reply:
Ok, I will think how tot change it. Can I use special class for this "header-title" for now?